Tests/Unit/Core/TransactionHandlerTest.php

```diff
@@ -58,6 +60,13 @@ protected function dbData()
                     ['transaction 2', 'oxid 1', 2, 'transaction 2', 'transaction 1', 'reserve', 'capture-authorization', 'success', 40, 'EUR'],
                     ['transaction 3', 'oxid 1', 2, 'transaction 3', 'transaction 1', 'reserve', 'capture-authorization', 'closed', 40, 'EUR'],
                     ['transaction 4', 'oxid 1', 2, 'transaction 4', 'transaction 3', 'reserve', 'refund-capture', 'closed', 40, 'EUR'],
+                    ['transaction 5', 'oxid 2', 3, 'transaction 5', null, 'pay', 'debit', 'success', 33.8, 'EUR'],
```

Comment:
please also add a test from very big numbers and numbers like `*.4`,  `*.5`, `*.6`, and something like `*.999`. Also please add a test with like HUF a which has quite big values.

Reply:
👍